project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val Scala212Version = "2.12.8"
 
   val Sbt013Version = "0.13.18"
-  val Sbt1Version = "1.3.3"
+  val Sbt1Version = "1.4.7"
```

Comment:
We haven't updated the sbt version so that we can be sure that the plugin will work will all versions `1.3.0+` Is this needed for fix? Alternatively we could release the plugin with 1.3.0, but I am not sure if that is possible.

Reply:
Sbt can only work with version 1.4.5+ on Apple M1 (https://github.com/sbt/sbt/issues/6187). If I didn't change the version, bloop can not build on my machine.

Reply:
em... It seems not the version in "build.properties"